sui/its.js

```diff
@@ -5,13 +5,18 @@ const { addBaseOptions, addOptionsToCommands, getWallet, printWalletInfo, broadc
 const { bcs } = require('@mysten/sui/bcs');
 
 const SPECIAL_CHAINS_TAGS = {
-    ALL_EVM: 'all-evm', // All EVM chains that have InterchainTokenService deployed
+    ALL: 'all',
 };
 
+function getAllTrustedChains(config) {
```

Comment:
This could take the `trustedChains` list and apply specialization internally, and ensure that the names match a valid `axelarId`